### ISSUE TITLE
Images are no longer rendering. Looks like the image file format has been changed by Notion

### ISSUE THREAD

**Issue by pbteja1998** [NotionX/react-notion-x#428]
#### Description
Now, image urls are starting with `files.notion.so` for some reason and that case is not being handled by the `defaultMapImageUrl`.

Check the below notion page for example.

#### Notion Test Page ID
39192dc8661e4776a91cfa5eb37c9499

**Comment by james-freec** [comment]
I have the same problem, any one know how it fix?

**Comment by normdoow** [comment]
Yeah, I'm seeing this in some pages as well. But not all of them. May just be on new workspaces or something.

**Comment by transitive-bullshit** [comment]
If any of you find a fix, please let us know.

**Comment by normdoow** [comment]
> If any of you find a fix, please let us know.

Quick fix I found is using the source from the image block instead of the signed_url
<img width="641" alt="image" src="https://user-images.githubusercontent.com/21371266/216759162-09e90874-a58c-47ab-a999-0015387b9ad7.png">
But the signed url is needed because it doesn't expire right?

**Comment by transitive-bullshit** [comment]
Signed URLs expire; maybe we can just use the `source` directly now? That would be *amazing*

**Comment by normdoow** [comment]
Hot fix: https://github.com/NotionX/react-notion-x/pull/430

**Comment by normdoow** [comment]
We will watch this to see how the hot fix performs in other scenarios. Once we learn more about the new notion source we can make a better solution.

**Comment by pbteja1998** [comment]
Looks like now even the audio files have signed URLs starting with `file.notion.so`

Related: https://github.com/NotionX/react-notion-x/issues/441

**Comment by nyx-aespid** [comment]
Same root cause as a few of the signed-URL threads here (#626, #441): Notion's `/image/` redirect (and now `file.notion.so` for audio) hands back a presigned S3/Notion URL that expires after ~1 hour, so anything statically generated or cached points at dead links shortly after build.

The durable fix is to fetch the asset once server-side and re-serve it from your own storage/CDN instead of embedding Notion's redirect directly. marc7806/notion-api-cache is a free proxy that does exactly this, and there are DIY Cloudflare Workers + R2 guides if you'd rather self-host. We also run a small hosted version of that pattern (permanent cache, no infra to maintain) if you'd rather not run the Worker yourself — happy to share details if useful, otherwise the proxy/Worker route is a solid free path.
